rfcs/no_repeat_expected.md

```diff
@@ -0,0 +1,57 @@
+# RFC 119: Allow wptrunner to skip tests that run as expected
+
+## Summary
+
+Add a `--no-repeat-expected` flag to wptrunner to skip tests in subsequent
+repeat iterations once they run as expected.
+
+## Details
+
+### Rationale
+
+This flag is used with `--repeat` to implement a retry mechanism.
+Retries are intended for CI systems that should be robust to false positives
+that unnecessarily block developers:
+* Transient infrastructure issues (e.g., CPU utilization spike causing
+  timeouts).
+* Incomplete expectation data checked into the trunk of a browser vendor's
+  source control system.
+  The CI check might not surface statuses that should have been added.
+
+### Proposal
+
+When passed `--no-repeat-expected`, wptrunner will run all loaded tests in the
+initial repeat iteration.
+On subsequent iterations, tests that ran as expected in a previous iteration
```

Comment:
Do we actually need the `SKIP` status? I think it might be easier to just not generate a status for those tests (since `SKIP` usually implies that the test was actively disabled, not that it was just not selected for the current run).

Reply:
Thanks for the info. No need for `SKIP`s then.